README.md

```diff
@@ -99,16 +99,19 @@ seatmapApi.highlightSection('upper end zone 232')
+| `onSelection`         |          | `function`                     |                                                                                                                                                                                                                                                    | A function which will be called by Tevomaps when a section of the map has been clicked. It will pass as arguments an array of all currently selected section IDs and expect nothing back.<br><br>`onSelection: function (sectionIds) {`<br>&nbsp;&nbsp;&nbsp;&nbsp;`console.log(sectionIds); //=> ['id-1','id-2']`<br>`}`<br><br>*Note: This method is also called when a section is deselected. If all sections are deselected, the `sectionIds` array will be empty.*<br><br>*Note: This method will always return lower case section names, you will need to take that in consideration when you do string comparisons.* |
+| `selectedSections`    |          | `string[]`                     | `[]`                                                                                                                                                                                                                                               | An array of section IDs for the map to initially highlight by default when it is rendered.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `ticketGroups`        |          | [`TicketGroup[]`](#interfaces) | `[]`                                                                                                                                                                                                                                               | The available ticket groups for this event.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
I would leave the `ticketGroups` description the way it is, especially the callout for `include_tevo_section_mappings`. That copy was decided on by broker services, and they want notes about the extra flag and the ticket groups API.

Reply:
Oh snap! My editor must have cleared the existing description when I formatted before editing this file. I saw this as blank and wanted to complete the config descriptions. Good catch!